frontend/src/metabase/dashboard/actions/parameters.ts

```diff
@@ -133,12 +173,13 @@ export const setParameterMapping = createThunkAction(
 
       dispatch(
         setDashCardAttributes({
-          id: dashcard_id,
+          id: dashcardId,
           attributes: {
             parameter_mappings: getParameterMappings(
-              dashcard,
-              parameter_id,
-              card_id,
+              // TODO remove type casting when getParameterMappings is fixed
+              dashcard as QuestionDashboardCard,
```

Comment:
However, not here, because I believe that [`getParameterMappings`](https://github.com/metabase/metabase/blob/hk-dashboard-actions-parameters-ts/frontend/src/metabase/dashboard/actions/auto-wire-parameters/utils.ts#L131-L165) currently _incorrectly_ expects only a `QuestionDashboardCard`.

@kulyk could you confirm that please? I am asking because you [changed this recently](https://github.com/metabase/metabase/pull/38444/files#diff-9d4407a615d5b08c4b7b4e07c6f83468444aa21d936c440b7e40b9d49559eab6R129).

TBH, I am not sure how to type it properly so I am leaving it outside of this PR.